### PR TITLE
fix remote repo urls

### PR DESCRIPTION
**Pull request by bernt-matthias**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/spot_detection_2d/.shed.yml [hunk 1]
```diff
@@ -6,4 +6,4 @@ description: Spot detection in 2D image sequence
 long_description: |
   This tool detects spots and measures the intensities in a 2D image sequence based on local intensity maxima.
 homepage_url: https://github.com/BMCV/galaxy-image-analysis
-remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/spot_detection_2d/
\ No newline at end of file
+remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tree/master/tools/spot_detection_2d/
```
Comment (bgruening) [inline tools/spot_detection_2d/.shed.yml:9]: this one does not look correct

File: tools/points_association_nn/.shed.yml [hunk 2]
```diff
@@ -6,4 +6,4 @@ description: Association of points in consecutive frames
 long_description: |
   This tool associates points in consecutive frames (slices) using the nearest neighbor algorithm.
 homepage_url: https://github.com/BMCV/galaxy-image-analysis
-remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/points_association_nn/
\ No newline at end of file
+remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tree/master/tools/points_association_nn/
```
Comment (bgruening) [inline tools/points_association_nn/.shed.yml:9]: this one as well

File: tools/overlay_images/.shed.yml [hunk 3]
```diff
@@ -6,4 +6,4 @@ description: Overlay two images
 long_description: |
   This tool overlays two image to visualize 1) image blending, 2) colocalization, and 3) a segmentation mask over an image.
 homepage_url: https://github.com/BMCV/galaxy-image-analysis
-remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/overlay_images/
\ No newline at end of file
+remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tree/master/tools/overlay_images/
```
Comment (bgruening) [inline tools/overlay_images/.shed.yml:9]: here

File: tools/3d_tensor_feature_dimension_reduction/.shed.yml [hunk 4]
```diff
@@ -6,4 +6,4 @@ description: Dimensionality reduction for features (channels) of 3D tensor data
 long_description: |
   This tool performs dimensionality reduction for features (channels) of 3D tensor data using UMAP. The results are saved as a multi-channel 32-bit TIFF image.
 homepage_url: https://github.com/BMCV/galaxy-image-analysis
-remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/3d_tensor_feature_dimension_reduction/
\ No newline at end of file
+remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tree/master/tools/3d_tensor_feature_dimension_reduction/
```
Comment (bernt-matthias) [inline tools/3d_tensor_feature_dimension_reduction/.shed.yml:9]:
```suggestion
remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/3d_tensor_feature_dimension_reduction/
```

File: tools/curve_fitting/.shed.yml [hunk 5]
```diff
@@ -6,4 +6,4 @@ description: Polynomial curve fitting to data points
 long_description: |
   This tool fits (1st- or 2nd-degree) polynomial curves to data points.
 homepage_url: https://github.com/BMCV/galaxy-image-analysis
-remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/curve_fitting/
\ No newline at end of file
+remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tree/master/tools/curve_fitting/
```
Comment (bernt-matthias) [inline tools/curve_fitting/.shed.yml:9]:
```suggestion
remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tree/master/tools/curve_fitting/
```

File: tools/overlay_images/.shed.yml (hunk 3 above)
Comment (bernt-matthias) [inline tools/overlay_images/.shed.yml:9]:
```suggestion
remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tree/master/tools/overlay_images/
```

File: tools/spot_detection_2d/.shed.yml (hunk 1 above)
Comment (bernt-matthias) [inline tools/spot_detection_2d/.shed.yml:9]:
```suggestion
remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tree/master/tools/spot_detection_2d/
```

File: tools/points_association_nn/.shed.yml (hunk 2 above)
Comment (bernt-matthias) [inline tools/points_association_nn/.shed.yml:9]:
```suggestion
remote_repository_url: https://github.com/BMCV/galaxy-image-analysis/tree/master/tree/master/tools/points_association_nn/
```

**Comment by bgruening** [comment]
I will merge and kill CI ok? I don't think we need to risk some TS tool updates. Or do we know it will just update description?

**Comment by bernt-matthias** [comment]
I'm not sure. Just kill CI. Or maybe `[skip-ci]` in the merge message?

**Pull request merged**